kuma/dashboards/forms.py

```diff
@@ -13,6 +13,11 @@
     ('week', _('Week')),
     ('month', _('30 days')),
 ]
+AUTHOR_CHOICES = [
+    ('All Authors', _('All Authors')),
```

Comment:
better to have the choice with underscore like ``('all_authors', _('All Authors')),``